docs/package.json

```diff
@@ -12,13 +12,13 @@
     "clear": "docusaurus clear"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.3.1",
-    "@docusaurus/plugin-google-gtag": "^2.3.1",
-    "@docusaurus/plugin-sitemap": "^2.3.1",
-    "@docusaurus/preset-classic": "^2.3.1",
-    "@mdx-js/react": "^1.6.22",
-    "@vercel/analytics": "^0.1.11",
-    "clsx": "^1.2.1",
+    "@docusaurus/core": "^2.4.1",
+    "@docusaurus/plugin-google-gtag": "^2.4.1",
+    "@docusaurus/plugin-sitemap": "^2.4.1",
+    "@docusaurus/preset-classic": "^2.4.1",
+    "@mdx-js/react": "^2.3.0",
```

Comment:
MDX v2 only works with Docusaurus v3, that's probably the problem.

Also we don't maintain the typedoc plugin on the Docusaurus repo, if there are issues on api pages I'm not sure I'll be able to help much on that, I don't really know how this plugin works 😅

Reply:
How do I get the full error logs/stacktrace? Is there a verbose param?

Reply:
Hmm I think I fixed that but maybe it's only in v3 or latest v2?

Not sure why it's so complicated to get full stacktraces printed 😅 this has always been annoying to me.